components/base/client_test.go

```diff
@@ -142,6 +155,24 @@ func TestClient(t *testing.T) {
 		test.That(t, resp["command"], test.ShouldEqual, testutils.TestCommand["command"])
 		test.That(t, resp["data"], test.ShouldEqual, testutils.TestCommand["data"])
 
+		// Spin
```

Comment:
If each of these sections tests a different thing, should it be a different invocation of `t.Run`? I don't feel strongly either way; keep as-is if that's what you prefer.

Reply:
changed, will test with a race and see if I'll commit that